cohorts/plot.py

```diff
@@ -22,7 +22,38 @@
 from sklearn.metrics import roc_curve
 from .model import bootstrap_auc
 
-def stripboxplot(x, y, data, ax=None, **kwargs):
+def vertical_percent(plot, percent=0.1):
+    plot_bottom, plot_top = plot.get_ylim()
+    return percent * (plot_top - plot_bottom)
+
+def set_min_y_from_data(y, data, plot):
+    """
+    If data is >= 0, don't show negative axis values.
+    """
+    min_y_val = data[y].min()
+    if min_y_val >= 0:
+        # Add a bit of padding so that we don't cut the plot off in an ugly way
+        extra_space = -1 * vertical_percent(plot, 0.05)
+        plot.set_ylim(bottom=extra_space)
```

Comment:
What I've done sometimes is turn off the bottom and top most tick label to have the extra space but not display negatives.

```
    yticks = plot.yaxis.get_major_ticks()
    yticks[0].set_visible(False)
    yticks[-1].set_visible(False)
```

Setting the max might be nice too, since sometimes we want things between 0 - 100%